GVRf/Framework/framework/src/main/java/org/gearvrf/GVRAndroidResource.java

```diff
@@ -295,9 +295,15 @@ public GVRAndroidResource(GVRContext context, URL url,
      */
     public synchronized final InputStream getStream() {
         if (streamState != StreamStates.OPEN) {
-            openStream();
+            try {
```

Comment:
There is a lot of code that doesn't expect this method to return null. Please let the exception escape getStream. There will be methods that will need to be updated. It is better knowing the root cause of the error instead of trying to deal with a vague null reference value.